addons/account/views/product_view.xml

```diff
@@ -9,16 +9,18 @@
             <field name="arch" type="xml">
                 <page string="Sales" position="after">
                     <page string="Accounting" groups="account.group_account_invoice">
-                        <group>
-                            <label for="categ_id" string="Internal Category"/>
-                            <div><field name="categ_id" colspan="3" nolabel="1"/></div>
+                        <group name="product_category">
+                            <group name="category">
+                                <label for="categ_id" string="Internal Category"/>
+                                <div><field name="categ_id" colspan="3" nolabel="1"/></div>
+                            </group>
                         </group>
                         <group name="properties">
-                            <group>
+                            <group name="sale_tax_part">
```

Comment:
i know names are always usefull when we inheriting views, but here no need to change that things for this task. it's useless change. here field name is enough to add something.